### ISSUE TITLE
Partitioning is missing in Structure page UI

### ISSUE THREAD

**Issue by sky93** [phpmyadmin/phpmyadmin#15276]
Hello,

I cannot find Partitioning feature in table structure tab. I'm using the latest version of PMA (4.8.5) but I can't find partitioning section anywhere. How can I bring partitioning feature back ?

![part](https://user-images.githubusercontent.com/8404511/57953739-36f06680-7906-11e9-94b1-c0477cbf8524.jpg)

**Comment by williamdes** [comment]
@sky93 Yes, strange ..
Thanks for reporting

**Comment by DanialV** [comment]
Same issue here any fix for this bug?

**Comment by williamdes** [comment]
@DanialV @sky93 This is not a bug, you do not have the partition plugin enabled.
See: https://demo.phpmyadmin.net/QA_4_8/tbl_create.php?server=1&db=opengis

run
```sql
SHOW PLUGINS
```
If you have `partition` an not partition option in the GUI comment on this issue please.

https://demo.phpmyadmin.net/QA_4_8/server_plugins.php?server=1
https://demo.phpmyadmin.net/QA_4_8/server_plugins.php

**Comment by sky93** [comment]
There's a problem here. In MariaDB when I run `SHOW PLUGINS` I can verify than `partition` is active and enabled. In MySQL I'm sure partition is enabled because I could create a table with partitioning:

```sql
CREATE TABLE ti (id INT, amount DECIMAL(7,2), tr_date DATE)
    ENGINE=INNODB
    PARTITION BY HASH( MONTH(tr_date) )
    PARTITIONS 6;
```

And I could verify partitions using this query:
```sql
SELECT PARTITION_NAME, TABLE_ROWS FROM INFORMATION_SCHEMA.PARTITIONS WHERE TABLE_NAME='ti'
```

So partitioning is enabled but doesn't show up in  `SHOW PLUGINS`. I guess this problem is only for MySQL 8.0 because I tired   `SHOW PLUGINS` in MariaDB and Mysql 5 and it's showing partition.

I guess partitioning is enabled in https://demo.phpmyadmin.net/QA_4_8 (Mysql) server too but not showing up in `SHOW PLUGINS`

**Comment by williamdes** [comment]
@sky93 Thanks !
MySQL 8.0 seems to have some native handling for partitions

**Comment by williamdes** [comment]
```sql
CREATE TABLE ti (id INT, amount DECIMAL(7,2), tr_date DATE)
    ENGINE=MyISAM
    PARTITION BY HASH( MONTH(tr_date) )
    PARTITIONS 6;
```
MySQL 5.7: `Warning: #1287 The partition engine, used by table 'public.ti', is deprecated and will be removed in a future release. Please use native partitioning instead.`

MySQL 8.0: `Error: #1178 - The storage engine for the table doesn't support native partitioning`

**Comment by williamdes** [comment]
MariaDB is not affected.

**Comment by williamdes** [comment]
https://dev.mysql.com/doc/refman/8.0/en/partitioning.html

> It is not possible to disable partitioning support by the InnoDB storage engine.

**Comment by sky93** [comment]
> ```sql
> MyISAM
> ```

Yes in MySQL 8.0 MyISAM engine doesn't support partitioning. It seems partitioning is enabled by default in MySQL 8.0.

**Comment by williamdes** [comment]
@DanialV @sky93 Can you please apply the patch 09d20e8c8d89618a7865b7b7af6a7f76acf3aa6a and tell me if everything looks okay

Tested on https://phpmyadmin-pr-7.wdes.eu.org https://phpmyadmin-pr-6.wdes.eu.org (public/public) and was OK

If you approve the change it goes directly for next release (4.8.6) :tada: :rocket:

**Comment by sky93** [comment]
Yes it's now fixed 🎉😍 Thank you so much for the fix.

**Comment by williamdes** [comment]
The fix will be part of `4.8.6`
Thanks for helping !
